assets/js/googlesitekit/modules/datastore/settings.js

```diff
@@ -0,0 +1,126 @@
+/**
+ * `core/modules` data store settings
+ *
+ * Site Kit by Google, Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import invariant from 'invariant';
+
+/**
+ * Internal dependencies
+ */
+import Data from 'googlesitekit-data';
+const { createRegistrySelector } = Data;
+
+// Actions
+const SET_MODULE_SETTINGS_PANEL_STATE = 'SET_MODULE_SETTINGS_PANEL_STATE';
+
+export const initialState = {
+	panelState: {},
+};
+
+export const actions = {
+	/**
+	 * Dispatches the module settings panel state action.
```

Comment:
Let's word this a bit more generically for consistency with most other actions 🙂 

```suggestion
	 * Sets the module settings panel state for a given module.
```

Reply:
Addressed.